hack/get-kubeconfig.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# Copyright 2020 The KubeCarrier Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu -o pipefail
+
+# Set OS specific values.
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    BASE64_DECODE_FLAG="-d"
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+    BASE64_DECODE_FLAG="-D"
+elif [[ "$OSTYPE" == "linux-musl" ]]; then
+    BASE64_DECODE_FLAG="-d"
+else
+    echo "Unknown OS ${OSTYPE}"
+    exit 1
+fi
+
+kubectl apply -f - <<EOF >> /dev/null
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubecarrier-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubecarrier-sa
+  namespace: kubecarrier-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
use `config/serviceCluster/role.yaml`  , don't hardcode these kind of stuff in files.

Reply:
done